app/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping.md

```diff
@@ -37,9 +37,13 @@ to identify which admin value and role name to map from the IdP to {{site.base_g
 The `admin_claim` value specifies which IdP username value should map to Kong Manager. 
 The username and password are required for the user to log into the IdP.
 
-The `authenticated_groups_claim` value specifies which IdP claim should be assigned to the
-specified {{site.base_gateway}} user. The exact value depends on your IdP -- for example, Okta 
-configures claims for `groups`, and other IdPs might configure them as `roles`.
+The `authenticated_groups_claim` value specifies which IdP claim should be used to assign {{site.base_gateway}} roles to the
+specified {{site.base_gateway}} admin. 
+
+- The group claim name depends on your IdP -- for example, Okta configures claims for `groups`, and other IdPs might configure them as `roles`. 
+- The group claim value need to follow the format `<workspace_name>:<role_name>` for dynamic role assigning. 
+
+For example, if we set `"authenticated_groups_claim": ["groups"]` and `groups:["default:super-admin"]` is returned on admins' id tokens, these admins gets `super-admin` role in `default` workspace. 
```

Comment:
```suggestion
For example, if `"authenticated_groups_claim": ["groups"]` is specified, and in the IdP `groups:["default:super-admin"]` is specified, the administrators specified in `admin_claim` are assigned to the super-admin role in the default {{site.base_gateway}} workspace. 
```

Reply:
I wrote `returned on admins' id tokens` because I want the users to understand the mapping only works if their ID token has this `claim:value` pair. (currently we are using `/userinfo` but whatever is on `/userinfo` should be listed on `id_token` anyway)

Only set `groups:["default:super-admin"]` on IdP does not mean `groups:["default:super-admin"]` will be returned on token or `/userinfo`. If this value is not found, the mapping will fail.

Reply:
Yeah, I get this. But it's too much detail on the IdP side, when this page is about the Kong Manager config. Kong users need to check the JWT on the IdP side, but they shouldn't have to do it on the Kong Manager side and if they do, we should probably have a troubleshooting page for this in the KB instead of putting it in the docs. Make sense? If the `claim:value` pair doesn't show up in the token, there are non-Kong issues to deal with ... And if we need to explain at this level of detail, it probably belongs on the OIDC config reference page, not here.

Reply:
Interesting point. Do we have a  troubleshooting page or config reference page for OIDC at the moment? If we have, I think it makes sense to put it there.

`If the claim:value pair doesn't show up in the token, there are non-Kong issues to deal with ...` I agree it is not Kong's problem. However when customer can't use it successfully, they will say it is Kong's problem.

The reason I am writing this with so much detail because we know what customer's pain points are. You will be surprised how many cases we got are related to Kong manager OIDC integration. (Some of them are directly related to claim not in token) IMO there is nothing wrong to provide a bit more detail just to help customer understand what they need to do.

Reply:
I am so sorry it's taken me so long to get back to this PR. This extra information is super helpful, and I understand a lot better. I've put it back in, reworded for simplicity and clarity (I hope). Maybe Kong users are as familiar with IdPs and JWTs as you are? But in my experience elsewhere it's not always the case, so I tried to make things simpler.